department-of-veterans-affairs/va.gov-cms
# VACMS-2999: Updated Entity Browser permissions for Content based r…

…oles to allow access to Benefit Hub Contact Information and VA Benefit Hubs pages

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/2999

## Testing done


## Screenshots


## QA steps

As a content admin/publisher
1. Login
2. Navigate to any content type that has the VA Benefit Hubs field or the Need more help? Contact Information entity reference revision (IE. step-by-step)
3. Select the "Select Benefit Hub(s)" button, confirm the modal with the list of benefit hubs appears and not access restrictions exist.
4. Perform the same test on the "Add Benefit Hub Contacts" button.
5. Confirm you can save the node with no issues. 

## Definition of Done
